docs/en/advanced_guides/datasets.md

```diff
@@ -1 +1,247 @@
-# Datasets
+# Dataset
+
+Due to the Dataset class in each OpenMMLab codebases have some common characteristics and requirements, such as efficient internal data storage format, support for the concatenation of different datasets, dataset repeated sampling,
+MMEngine implements `BaseDataset` which provides some basic interfaces and implements some DatasetWrappers with the same interfaces. The Dataset class of each OpenMMLab downstream codebase would inherit this base dataset class and add customization methods.
+
+The basic function of the MMEngine `BaseDataset` is loading the dataset information which has two categories.
+One is meta information, which represents the information related to the dataset itself and sometimes needs to be obtained by the model or other external components.
+The other is data information, which defines the file path and corresponding label information of specific data info.
```

Comment:
it needs modification  `which defines the file path and corresponding label information of specific data info.`